### PR TITLE
pkp/pkp-lib#2604 pt_BR locale update

### PR DESCRIPTION
**Pull request by ronniebrito**

### CODE REVIEW

**Comment by mtub** [comment]
pkp/pkp-lib#2604

File: locale/pt_BR/admin.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
 	<message key="admin.auth.enableCreateUsers">Habilitar criação de usuários (caso suportado pelo plugin de autenticação!). Usuários criados no sistema serão automaticamente incluídos na fonte de autenticação remota, caso não existam ainda. Inclusive, usuários criados através do cadastro de usuários também serão incluídos automaticamente na fonte remota, caso esta seja a fonte de autenticação padrão.</message>
 	<message key="admin.systemVersion">Versão do OJS</message>
 	<message key="admin.systemConfiguration">Configuração do OJS</message>
-	<message key="admin.systemConfigurationDescription"><![CDATA[Configurações do OJS no documento <tt>config.inc.php</tt>.]]></message>
+	<message key="admin.systemConfigurationDescription"><![CDATA[Configurações do OJS no arquivo <<config.inc.php>>.]]></message>
```
Comment (mtub) [inline locale/pt_BR/admin.xml]: please add <tt>...</tt> again

File: locale/pt_BR/default.xml [hunk 2]
```diff
@@ -42,13 +42,13 @@ Siga o seguinte protocolo para fazer revisões eletrônicas em seu manuscrito:
 &nbsp; Para cada sugestão rejeitada, insira o texto original novamente e aplique <strong>negrito</strong>.
 
 <strong>Realizando Inserções e Remoções</strong>
-&nbsp; Indique inserções novas em <strong>negrito</strong>.
-&nbsp; Substitua seções removidas por: <strong>[texto removido]</strong>.
-&nbsp; Caso remova uma frase ou mais, indique com um aviso (ex.: <strong>[removidas 2 frases]</strong>).
+ Indique inserções novas em <strong>negrito</strong>.
+ Substitua seções removidas por: <strong>[texto removido]</strong>.
+ Caso remova uma frase ou mais, indique com um aviso (ex.: <strong>[removidas 2 frases]</strong>).
```
Comment (mtub) [inline locale/pt_BR/default.xml]: These are not very important, but the &nbsp; are included in the en_US version and I'd rather have the same everywhere...

File: locale/pt_BR/default.xml [hunk 3]
```diff
@@ -42,13 +42,13 @@ Siga o seguinte protocolo para fazer revisões eletrônicas em seu manuscrito:
...
 
 <strong>Respondendo a dúvidas para o autor (DAs)</strong>
 &nbsp; Mantenha todas as DAs intactas e em negrito dentro do texto. Não as remova.
-&nbsp; Para responder a uma DA, insira um comentário após a dúvida. Delimite comentários da seguinte forma:
+ Para responder a uma DA, insira um comentário após a dúvida. Delimite comentários da seguinte forma:
 <strong>[Comentário: ]</strong>
```
Comment (mtub) [inline locale/pt_BR/default.xml]: restore &nbsp;?

File: locale/pt_BR/locale.xml [hunk 4]
```diff
@@ -130,7 +130,7 @@
 	<message key="submission.submissionEditing">Edição da submissão</message>
 	<message key="submission.changeSection">Alterar para</message>
 	<message key="submission.editorDecision">Decisão Editorial</message>
-	<message key="submission.round"><![CDATA[Rodada&nbsp;{$round}]]></message>
+	<message key="submission.round"><![CDATA[Rodada {$round}]]></message>
```
Comment (mtub) [inline locale/pt_BR/locale.xml]: Why are you removing this?

File: locale/pt_BR/locale.xml [hunk 5]
```diff
@@ -564,7 +564,7 @@ Ativo indica avaliação em consideração ou andamento.]]></message>
 
 	<!-- Interstitial page for non-PDF non-HTML galleys -->
 	<message key="article.nonpdf.title">Baixar artigo</message>
-	<message key="article.nonpdf.note"><![CDATA[O documento selecionado deverá ser baixado automaticamente. Em caso negativo, clique no atalho abaixo. Note que o documento pode requerer um visualizador especial ou plugin para ser executado.<br /><br />Caso o documento não seja baixado automaticamente, <a href="{$url}">clique aqui</a>.]]></message>
+	<message key="article.nonpdf.note"><![CDATA[O documento selecionado deverá ser baixado automaticamente. Em caso negativo, clique no atalho abaixo. Note que o documento pode requerer um visualizador especial ou plugin para ser executado.<br/><br/>Caso o documento não seja baixado automaticamente, <a href="{$url}">clique aqui</a>.]]></message>
```
Comment (mtub) [inline locale/pt_BR/locale.xml]: again, you are changing tags from the en_US version - <br /> should be restored

File: plugins/generic/usageStats/locale/pt_BR/locale.xml [hunk 6]
```diff
@@ -25,4 +25,17 @@
 		<li>Navegador</li>
 		</ul>
 		<p>Os dados coletados são usados apenas para propósitos de avaliação. Nenhum endereço de IP é mapeado a IDs de usuário. É tecnicamente impossível rastrear um conjunto específicos de dados a um IP específico.</p>]]></message>
+  <message key="plugins.generic.usageStats.settings.statsDisplayOptions.siteWide">&lt;h3&gt;Informações gerais sobre privacidade&lt;/h3&gt;
```
Comment (mtub) [inline plugins/generic/usageStats/locale/pt_BR/locale.xml]: this and the following lines: please restore HTML, your editors seems to have masked your HTML entities

File: plugins/importexport/datacite/locale/pt_BR/locale.xml [hunk 7]
```diff
@@ -25,4 +25,7 @@
 {$scriptName} {$pluginName} register [journal_path] {issues|articles|galleys} objectId1 [objectId2] ...
 ]]></message>
 	<message key="plugins.importexport.datacite.senderTask.name">Dados DataCite do remetente</message>
+  <message key="plugins.importexport.datacite.settings.description">Caso deseje registrar DOIs no DataCite, entre em contato com o Gerente por meio da &lt;a href="http://datacite.org/contact" target="_blank"&gt;página do DataCite&lt;/a&gt;, que irá encaminá-lo ao seu Representante DataCite local. Uma vez estabelecido o relacionamento com a organização representante, será providenciado o acesso ao serviço  DataCite para a cunhagem de identificadores persistente (DOIs) e registro de metadados associados. Mesmo que não possua um login e senha, será possível exportar dados no formato XML  DataCite, porém não poderá registrar seus DOIs no DataCite por meio do OJS. Note que a senha será gravada como texto puro, isto é, não encriptada, devido aos requisitos de registro do serviço DataCite.</message>
```
Comment (mtub) [inline plugins/importexport/datacite/locale/pt_BR/locale.xml]: same as above: &lt;li&gt; -> `<li>` etc.

File: plugins/importexport/medra/locale/pt_BR/locale.xml [hunk 8]
```diff
@@ -40,4 +40,11 @@
 {$scriptName} {$pluginName} export [xmlFileName] [journal_path] {issues|articles|galleys} objectId1 [objectId2] ...
 {$scriptName} {$pluginName} register [journal_path] {issues|articles|galleys} objectId1 [objectId2] ...
 ]]></message>
+  <message key="plugins.importexport.medra.settings.form.exportIssuesAs.label">NB: DOIs designados a artigos serão exportados para o mEDRA como &lt;a href="http://www.medra.org/en/metadata_td.htm" target="_blank"&gt;'trabalhos'&lt;/a&gt;. DOIs designados a composições publicadas serão exportadas como &lt;a href="http://www.medra.org/en/metadata_td.htm" target="_blank"&gt;'manifestações'&lt;/a&gt;.</message>
```
Comment (mtub) [inline plugins/importexport/medra/locale/pt_BR/locale.xml]: same as above: &lt;a href&gt;-> `<a href>` etc.

File: plugins/importexport/medra/locale/pt_BR/locale.xml [hunk 9]
```diff
@@ -40,4 +40,11 @@
...
+  <message key="plugins.importexport.medra.senderTask.name">Uso:
+{$scriptName} {$pluginName} export [xmlFileName] [journal_path] {issues|articles|galleys} objectId1 [objectId2] ...
+{$scriptName} {$pluginName} register [journal_path] {issues|articles|galleys} objectId1 [objectId2] ...
+</message>
+  <message key="plugins.importexport.medra.settings.form.automaticRegistration.description">NB: DOIs designados a artigos serão exportados para o mEDRA como &lt;a href="http://www.medra.org/en/metadata_td.htm" target="_blank"&gt;'trabalhos'&lt;/a&gt;. DOIs designados a composições publicadas serão exportadas como &lt;a href="http://www.medra.org/en/metadata_td.htm" target="_blank"&gt;'manifestações'&lt;/a&gt;.</message>
```
Comment (mtub) [inline plugins/importexport/medra/locale/pt_BR/locale.xml]: same as above: &lt;a href&gt;-> `<a href>` etc.

File: plugins/importexport/medra/locale/pt_BR/locale.xml [hunk 10]
```diff
@@ -40,4 +40,11 @@
...
+  <message key="plugins.importexport.medra.settings.form.testMode.description">NB: DOIs designados a artigos serão exportados para o mEDRA como &lt;a href="http://www.medra.org/en/metadata_td.htm" target="_blank"&gt;'trabalhos'&lt;/a&gt;. DOIs designados a composições publicadas serão exportadas como &lt;a href="http://www.medra.org/en/metadata_td.htm" target="_blank"&gt;'manifestações'&lt;/a&gt;.</message>
```
Comment (mtub) [inline plugins/importexport/medra/locale/pt_BR/locale.xml]: same as above: &lt;a href&gt;-> `<a href>` etc.

File: locale/pt_BR/default.xml (hunk 2 above)
Comment (ronniebrito) [inline locale/pt_BR/default.xml]: You are right, will push a new version with the &nbsp

File: locale/pt_BR/locale.xml (hunk 4 above)
Comment (ronniebrito) [inline locale/pt_BR/locale.xml]: &nbsp; looks noisy for non IT translators... removing the &nbsp; did not compromised final results...

**Comment by mtub** [comment]
@ronniebrito , the submodule updates (pkp-lib and pdfJsView) are interfering with merging this. I will cherry-pick your locale updates and commit them from my account.

**Pull request closed without merging**